vscode-powershell.build.ps1

```diff
@@ -33,6 +33,7 @@ task ResolveEditorServicesPath -Before CleanEditorServices, BuildEditorServices,
     }
     else {
         $script:psesRepoPath = Resolve-Path $script:psesRepoPath
+        gci $env:BUILD_SOURCESDIRECTORY
```

Comment:
Should this be here? If so, can it be `Get-ChildItem` instead?

Reply:
```suggestion
```